apps/system/js/browser_context_menu.js

```diff
@@ -56,6 +56,9 @@
         return;
       }
 
+      evt.preventDefault();
+      evt.stopPropagation();
+
```

Comment:
can you add a small unit test?
we already have the plumbing in place to assert on `defaultPrevented` and we can to the same for `stopPropagation`